bolt-sidecar/src/state/execution.rs

```diff
@@ -123,6 +124,12 @@ impl ValidationError {
     }
 }
 
+impl From<pricing::PricingError> for ValidationError {
+    fn from(err: pricing::PricingError) -> Self {
+        ValidationError::Internal(format!("Pricing error: {}", err))
+    }
+}
```

Comment:
I think some validation in `validate_request()` is duplicated in `calculate_min_priority_fee()` so we might not need this after some deduplication